cli/command/network/create.go

```diff
@@ -63,6 +66,12 @@ func newCreateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.SetAnnotation("attachable", "version", []string{"1.25"})
 	flags.BoolVar(&opts.ingress, "ingress", false, "Create swarm routing-mesh network")
 	flags.SetAnnotation("ingress", "version", []string{"1.29"})
+	flags.BoolVar(&opts.swarm, "swarm", false, "Create a swarm network")
```

Comment:
If I saw this as a user, I would think I had to specify this flag when creating an overlay network I intended to use with Swarm, but I assume that's not actually the case.